File: app/src/main/java/com/boardgamegeek/io/model/HotGame.java
package com.boardgamegeek.io.model;

import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Path;
import org.simpleframework.xml.Root;

@Root(name = "item")
public class HotGame {
	@Attribute public int id;

	@Attribute public int rank;

	@Attribute(name = "value")
	@Path("thumbnail") public String thumbnailUrl;

	@Attribute(name = "value")
	@Path("name") public String name;

	@Attribute(name = "value")
	@Path("yearpublished") public int yearPublished;
}

File: app/src/main/java/com/boardgamegeek/model/HotGame.kt
package com.boardgamegeek.model

data class HotGame(
    val rank: Int = 0,
    val id: Int = 0,
    val name: String = "",
    val thumbnailUrl: String = "",
    val yearPublished: Int = YEAR_UNKNOWN
) {
    companion object {
        const val YEAR_UNKNOWN = Game.YEAR_UNKNOWN
    }
}
